sqfentity/lib/sqfentity.dart

```diff
@@ -257,10 +257,22 @@ class SqfEntityProvider extends SqfEntityModelBase {
   }
 
   Future<BoolResult> updateBatch(
-      QueryParams params, Map<String, dynamic> values) async {
+      QueryParams params, Map<String, dynamic> values,
+      [bool useHook = true]) async {
     final result = BoolResult(success: false);
     if (openedBatch[_dbModel!.databaseName!] == null) {
       try {
+        if (_dbModel!.postSaveAction != null) {
+          if (useHook) {
+            final primaryKey = params.whereString?.split('=?')[0];
+            final primaryKeyValue = params.whereArguments?.first;
+
+            values.putIfAbsent(primaryKey!.trim(), () => primaryKeyValue);
+
+            await _dbModel!.postSaveAction!(_tableName!, values, 'UPDATE');
+          }
+        }

```

Comment:
This is a `postSaveAction` and should only be called after the "update" function is triggered below and not before